src/app/recover.rs

```diff
@@ -0,0 +1,20 @@
+use futures::Stream;
+use linkerd2_error::Error;
+pub use linkerd2_error::Recover;
+pub use linkerd2_exp_backoff::{ExponentialBackoff, ExponentialBackoffStream};
+
+pub fn always(backoff: ExponentialBackoff) -> Always {
```

Comment:
Any reason this is a function and not `fn new` method on an `Always` impl?

Reply:
everything else in our stack builders looks like this, because it's prettier